src/EprPrnIntegration.Api/UpdateProducersFunction.cs

```diff
@@ -2,27 +2,33 @@
 using EprPrnIntegration.Common.Constants;
 using EprPrnIntegration.Common.Mappers;
 using EprPrnIntegration.Common.Models;
+using EprPrnIntegration.Common.Models.Queues;
 using EprPrnIntegration.Common.RESTServices.BackendAccountService.Interfaces;
+using EprPrnIntegration.Common.Service;
 using Microsoft.AspNetCore.Http;
 using Microsoft.Azure.Functions.Worker;
 using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.Logging;
 
 namespace EprPrnIntegration.Api;
 
-public class UpdateProducersFunction(IOrganisationService organisationService, INpwdClient npwdClient, 
-    ILogger<UpdateProducersFunction> logger, IConfiguration configuration)
+public class UpdateProducersFunction(
+    IOrganisationService organisationService,
+    INpwdClient npwdClient,
+    ILogger<UpdateProducersFunction> logger,
+    IConfiguration configuration,
+    IServiceBusProvider serviceBusProvider)
 {
     [Function("UpdateProducersList")]
-    public async Task Run([HttpTrigger(AuthorizationLevel.Anonymous, "get")] HttpRequest req)
+    public async Task Run([TimerTrigger("%UpdateProducersTrigger%")] TimerInfo myTimer)
     {
         logger.LogInformation($"UpdateProducersList function executed at: {DateTime.UtcNow}");
 
-        int startHour = GetStartHour(configuration["UpdateProducersStartHour"]);
-
-        var toDate = DateTime.Today.AddHours(startHour);
-        var fromDate = toDate.AddDays(-1);
+        var deltaRun = await DeltaSyncExecution();
 
+        var toDate = DateTime.UtcNow;
+        var fromDate = deltaRun.LastSyncDateTime;
+         
         logger.LogInformation($"Fetching producers from {fromDate} to {toDate}.");
```

Comment:
String interpolation $ is not as efficient in loggers, better log("{X}", x) than log($({x}");

Reply:
done